### PR TITLE
Portfolio: preserve data in the browser

### PR DESCRIPTION
**Pull request by Nelofarzabi**
In this project, we are implementing this : 

- Use local storage to save user input.

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @Nelofarzabi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that is supposed to be pair-programming but has commits only from one student. Kinldy ensure every member of the group has at least one commit in the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -202,8 +202,8 @@ function formValidate(event) {
   } else { error.style.visibility = 'hidden'; }
 }
 form.addEventListener('submit', formValidate);
-const message = document.getElementById('message');
-const userName = document.getElementById('contact-name');
+const message = document.getElementByclass('message');
+const userName = document.getElementByclass('contact-name');
 
 function storeData() {
```
Comment (shahadat3669) [inline script.js:209]: - [ ] Well! Your UI looks visually appealing. However, according to the project requirements, when the user changes the content of any input field, the data should be saved to the local storage. Additionally, when the user loads the page, if there is any data stored in the local storage, the input fields should be pre-filled with that data. 🔄

Currently, it seems that this functionality is not working as expected in your project. Please review and update your code to ensure that the data is properly saved to the local storage and that the input fields are correctly pre-filled upon page load. Keep up the good work! 💪🌟

[Portfolio.webm](https://github.com/Nelofarzabi/My_Portfolio/assets/55840999/8f3331f8-3b64-481e-a8ed-5c179fc0ff4d)

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi Everyone, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Good PR Title
✅ Good PR Description
✅ Good use of GitHub Flows.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [Optional] Well done on adding the 'Future Features' section!👍 However, it would be good to include 1-3 potential features in this section. Remember in this section, you should outline your plans for the future development of the project. Include details about the features, functionalities, or improvements you intend to add. It should demonstrate your vision and give readers an idea of what to expect. Keep up the good work!

Please include 1-3 potential fractures in this section.

| Your | Professional |
| --- | --- |
| image | image |



Cheers and Happy coding!👏👏👏

## Have any doubt ❓


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## ⚠️ WARNING ⚠️
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Nelofarzabi 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Input fields value saved properly in local storage
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**